packages/terra-framework-site/src/app/common/menu/MenuList.scss

```diff
@@ -0,0 +1,32 @@
+.cm-container {
```

Comment:
Looks like stylelint is not hitting this file. We don't need to update it now, but we should get it configured to hit .scss files in terra-framework.